README.md

```diff
@@ -7,19 +7,252 @@ inspectpack
+
+So, we've got inefficient code that we discovered via a _manual_ inspection. Wouldn't it be nice to have a report that specifically highlighted problems like these with useful information?
+
+... enter the `DuplicatesPlugin`.
+
+#### Diagnosing duplicates
+
+##### Simple report
+
+With our plugin enabled in the standard configuration:
+
+```js
+new DuplicatesPlugin()
+```
+
+we get a summary report of the duplicates upon running the `webpack` command:
+
+```
+WARNING in Duplicate Sources / Packages - Duplicates found! ⚠️
+
+* Duplicates: Found a total of 2 similar files across 3 code sources (both identical + similiar) accounting for 703 bundled bytes.
+* Packages: Found a total of 1 packages with 2 resolved, 3 installed, and 3 depended versions.
+
+## bundle.js
+lodash (Found 2 resolved, 3 installed. Latest version 4.2.3.)
+  3.1.0 ~/one/~/lodash
+    scenario-new-webpack-new-npm-unflattened@* -> one@1.2.3 -> lodash@^3.0.0
+  3.1.0 ~/two/~/lodash
+    scenario-new-webpack-new-npm-unflattened@* -> two@2.3.4 -> lodash@^3.0.0
+  4.2.3 ~/lodash
+    scenario-new-webpack-new-npm-unflattened@* -> lodash@^4.1.0
+```
+
+Breaking down this report, we get a `webpack` "warning" emitted by default with an initial summary
+of the report.
+
+* The `Duplicates` summary looks at **what is in the `webpack` bundle**. It tells us there are 2 files that are not identical, but the same package file path (e.g `3.1.0` vs `4.2.3` for `lodash/index.js`) and that there are 3 code sources that end up in our final bundle (which includes _two_ for `4.2.3`). We also get a byte count for all the files at issue (`703` bytes), which presumably could roughly be cut by 2/3 if we could collapse to just **one** file to do the same thing.
+* The `Packages` summary looks at **what `npm` installed to `node_modules`**. This is the other "view" into our problems.
+    * _Terminology_: Let's dig in to what things mean here.
+        * **Resolved**: We have one package (`lodash`) that has 2 resolved versions (`3.1.0` and `4.2.3`). A "resolution" means that upon inspecting the dependency tree and what's in a registry source, these specific versions "match". The results may differ at a different point in time
+        * **Installed**: These are actual packages installed to the local disk. In our case, we have **three** installs for 2 resolutions because we place an identical version twice.
+        * **Depended**: These are the number of upstream packages that create a dependency from a unique path in the graph to a package. Put more concretely, in our case, three unique `package.json` files have an entry for `lodash`.
+            * _Note_: This is a bit of complicated assessment, since aside from the root `package.json` is, all of the rest of the dependency graph depends on what is resolved at the next level to give a dependent `package.json` and so on recusively.
```

Comment:
Nit – some grammar here 😉 
```suggestion
            * _Note_: This is a bit of a complicated assessment, since aside from the root `package.json` the rest of the dependency graph depends on what is resolved at the next level to give a dependent `package.json` and so on recusively.
```